dashboard/app/views/pd/workshop_mailer/teacher_cancel_receipt.html.haml

```diff
@@ -0,0 +1,28 @@
+:css
+  body {
+    font-family: Gotham, sans-serif;
+  }
+
+%p
+  You have canceled your registration for the Code.org
+  = @workshop.course
+  = @workshop.subject
```

Comment:
@breville  The `<course> <subject>` is common, but the rest of this only appears here. I'm don't think it's worth adding a function for this one case.

@tanyaparker added `workshop`